pkg/querytracker/querytracker.go

```diff
@@ -646,6 +646,45 @@ func GetPQSById(ctx *fasthttp.RequestCtx) {
 	ctx.SetStatusCode(fasthttp.StatusOK)
 }
 
+type PqsConfig struct {
+	PQSConfig string `json:"pqsConfig"`
+}
```

Comment:
keep naming consistent, we are calling it pqsEnabled, in other places, we should call this the same